src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/LSPConstants.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
```

Comment:
```suggestion
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

﻿using System;
```

Reply:
I ended up deleting this class and file since (as Taylor mentioned) we no longer need ContainedLanguageMarker

---
In reply to: [451969015](https://github.com/dotnet/aspnetcore-tooling/pull/2201#discussion_r451969015) [](ancestors = 451969015)